templates/tfengine/recipes/devops.hcl

```diff
@@ -32,6 +32,7 @@ schema = {
       description = <<EOF
         ID of parent GCP resource to apply the policy.
         Can be one of the organization ID or folder ID according to parent_type.
+        See <https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy> to learn more about resource hierarchy.
```

Comment:
At line 162 and 77 of this file (the descriptions for exists for two groups), you can say

"Whether or not the group exists already. It will be created it not."